### DISCUSSION TITLE
sensor.grid_active_power disappeared

### DISCUSSION THREAD

**Discussion by johanvh83** [wlcrs/huawei_solar#344] [General]
Hi, I've updated the integration and it seems a lot changed in the naming of the sensors.
All sensors have now kind of prefix 'inverter', 'battery' and 'power'.

The 'old' integration had a sensor.grid_active_power does anyone knows the new name?
Can't find it...

**Comment by wlcrs** [comment]
This will now be "Power meter Active power".

Renaming was due to https://developers.home-assistant.io/blog/2022/07/10/entity_naming/
